content/400-pulse/100-what-is-pulse.mdx

```diff
@@ -6,23 +6,48 @@ tocDepth: 3
 toc: true
 ---
 
-<TopBlock>
+## What is Prisma Pulse?
 
-[Prisma Pulse](https://www.prisma.io/data-platform/pulse) is managed database-event infrastructure that captures and distributes your database events to your application. It simplifies subscribing to type-safe data changes with an extended [Prisma Client](/orm/prisma-client) to power real-time functionality.
+[Prisma Pulse](https://www.prisma.io/data-platform/pulse) is managed infrastructure that captures databases events (i.e. create, update and delete operations) and distributes them to your applications.
 
-</TopBlock>
+It lets you subsribe to any changes happening in your database using [Prisma Client](/orm/prisma-client) to power **real-time use cases** in your applications, such as:
 
-## How Prisma Pulse works
+- chat and messaging
+- data syncing (e.g. into a search index)
+- social interaction and collaboration
```

Comment:
What is "social interaction"? How about roughly syncing up this list with what's on prisma.io/pulse? It's already very close :)

Reply:
> What is "social interaction"?

I was thinking about "like","follow", etc features that you often find in social media apps. Do you feel it's not clear enough and I should drop that altogether?

> How about roughly syncing up this list with what's on prisma.io/pulse? It's already very close :)

I did take inspiration from that list in the first place and took all the use case that I felt would work in this bullet list. I just looked at it again and added "onboarding emails" as another use case here: 

- chat and messaging
- sending onboarding emails
- data syncing (e.g. into a search index)
- social interaction and collaboration
- notifying users
- updating inventories
- managing payments
- ... and any other functionality that requires real-time updates

Are there any others you think should be added?

Reply:
I'd drop social interaction, or maybe rephrase to a more generic "updating user interfaces for changes" (or a smarter phrasing).

Real-time collaboration could then be it's own bullet.

And add "many" to "... and many other functionality"